openmrs-19/src/main/java/org/motechproject/openmrs19/service/impl/OpenMRSConfigServiceImpl.java

```diff
@@ -113,6 +124,27 @@ public Config getDefaultConfig() {
         return configs.getByName(configs.getDefaultConfigName());
     }
 
+    @Override
+    @Transactional
```

Comment:
This method doesn't interract with the database at all, I don't think we need transactional here.